build.ps1

```diff
@@ -192,15 +192,16 @@ function Get-InstallDir($Arch) {
   return "$ImageRoot\$ProgramFilesName\Swift"
 }
 
-$LibraryRoot = "$ImageRoot\Library"
-$ToolchainInstallRoot = "$(Get-InstallDir $HostArch)\Toolchains\$ProductVersion+Asserts"
-$PlatformInstallRoot = "$(Get-InstallDir $HostArch)\Platforms\Windows.platform"
-$RuntimeInstallRoot = "$(Get-InstallDir $HostArch)\Runtimes\$ProductVersion"
-$SDKInstallRoot = "$PlatformInstallRoot\Developer\SDKs\Windows.sdk"
+$ImageLibraryRoot = "$ImageRoot\Library"
+$ImageSwiftRoot = Get-ImageSwiftRoot $HostArch
+$ImageToolchainRoot = "$ImageSwiftRoot\Toolchains\$ProductVersion+Asserts"
+$ImagePlatformRoot = "$ImageSwiftRoot\Platforms\Windows.platform"
+$ImageRuntimeRoot = "$ImageSwiftRoot\Runtimes\$ProductVersion"
+$ImageSDKRoot = "$ImagePlatformRoot\Developer\SDKs\Windows.sdk"
```

Comment:
Why the renaming?  The behaviour of `BinaryRoot` should be similar to that of `DESTDIR` in autotools.

Reply:
I don't understand? The goal was to not name "Install" something that is not a CMake "Install" destination. Since we have "ImageRoot" as the parameter name, it follows that all of these would be called "Image" and that this distinguishes them from arch-specific "Arch.Install" directories.

Reply:
I previously understood that you preferred to keep the "Install" naming to only refer to cmake uses of the term.

Reply:
Yes (though I think that BinaryRoot is still the better term).  Once you swap the order, I think the renaming will make a bit more sense.  It just seemed like you were changing the variables but I missed the change in the name itself.